habitat/plan.sh

```diff
@@ -24,46 +25,63 @@ pkg_build_deps=(
 )
```

Comment:
Does it make sense to use `scaffolding-ruby` here?

Reply:
Unless something changed recently, the Ruby scaffolding worked well for rails, etc... but not for standalone ruby projects. Happy to consider a change in the future, but I'd like the focus of this PR to be on the automation of the existing package building vs. refactoring the package.